src/autoscaler/eventgenerator/generator/evaluator.go

```diff
@@ -70,27 +70,27 @@ func (e *Evaluator) doEvaluate(triggerArray []*model.Trigger) {
 			return
 		}
 		for _, appMetric := range appMetricList {
-			if appMetric.Value < 0 {
+			if appMetric.Value == nil {
 				e.logger.Info("should not send trigger alarm to scaling engine because there is nil-value metric", lager.Data{"trigger": trigger, "appMetric": appMetric})
```

Comment:
I know what the code is doing. You have marked this as an info message. I want to know what an operator is supposed to do when they see this message because they aren't going to understand it. The message describes what the code is doing not what the operator should be concerned about.